flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -412,18 +417,38 @@ private String operatorName(String suffix) {
 
       switch (writeMode) {
         case NONE:
+          if (!equalityFieldIds.isEmpty()) {
+            LOG.info("Distribute rows by equality fields in '{}' distribution mode", DistributionMode.NONE.modeName());
```

Comment:
Are so many info statements needed?

Reply:
Yes, I agree with steven on this [discussion ](https://github.com/apache/iceberg/pull/2898#discussion_r779281367) that we should indicate we distribute rows by what key in what distribution mode.

Reply:
There are quite a number of info logs that seem to repeat similar statements. Can these at least be merged somehow?

Reply:
I deleted some unnecessary logs and rewritten the statements. Now the statements of the logs will not be repeated. Please take a look again.